styles.css

```diff
@@ -60,13 +60,23 @@ nav a {
   padding: 0.8em 2em;
   /* text ------- [icon] */
   display: flex;
-  flex-direction: row-reverse;
   justify-content: space-between;
   align-items: center;
+  /* use as a parent for absolute positioning */
+  position: relative;
 }
 nav a.selected {
   background-color: rgba(50, 150, 220, 0.3);
 }
+nav a.folder::before {
+  content: "▶";
```

Comment:
what about `►` and `▼` (\u25ba and \u25bc respectively).

I think the disproportional size makes it look ugly when you switch pages (especially if we had multiple folders)

Reply:
Right, this is much better